surface_potential_analysis/kernel/solve.py

```diff
@@ -468,7 +590,7 @@ def get_noise_operators_real_isotropic_stacked_taylor_expansion(
     ).astype(np.complex128)
     operator_coefficients *= n_states
 
-    operators = get_operators_for_real_isotropic_noise(basis_x, n_terms=n + 1)
+    operators = get_operators_for_real_isotropic_noise(basis_x, n=n + 1)
 
     return {
         "basis": operators["basis"],
```

Comment:
make code work for system of arbitrary dimension, seems good to me, have they been tested and do they work?

Reply:
So it doesn't actually work - but I haven't broken 1d. This was mostly about making the code cleaner in preparation, and building out some visualizations that work in 2d